olp-cpp-sdk-core/include/olp/core/cache/CacheSettings.h

```diff
@@ -31,14 +31,20 @@ namespace cache {
  * @brief Options for opening the disk cache
  */
 enum OpenOptions : unsigned char {
-  Default = 0x00,
-  ReadOnly = 0x01,
-  CheckCrc = 0x02
+  Default = 0x00, /*!< Opens the disk cache in read-write mode */
+  ReadOnly = 0x01, /*!< Opens the disk cache in read only mode */
```

Comment:
```suggestion
  ReadOnly = 0x01, /*!< Opens the disk cache in read only mode */
```
Misaligned comments - run clang-format